figures/mpo-breakout-port-example.txt

```diff
@@ -0,0 +1,44 @@
+Editors' Note: add the port example in slide 9

```

Comment:
remove the notes here. Only the pictures are needed.

Reply:
Should be fixed with 77c6449bb96fc1fcdc3e975e1ee17d0a5bc07a34